include/Dialect/BGV/IR/BUILD

```diff
@@ -47,6 +47,12 @@ gentbl_cc_library(
             ],
             "BGVDialect.cpp.inc",
         ),
+        (
```

Comment:
Thanks! This actually reminds me to make a cleanup to create a bazel rule that does all the default creation for types, ops, and dialect. Similar to [`add_mlir_dialect` ](https://mlir.llvm.org/docs/Tutorials/CreatingADialect/#tablegen-targets)